src/OpenTelemetry.Extensions.Hosting/Implementation/TelemetryHostedService.cs

```
@@ -33,16 +33,9 @@ public TelemetryHostedService(IServiceProvider serviceProvider)
 
     public Task StartAsync(CancellationToken cancellationToken)
     {
-        try
-        {
-            // The sole purpose of this HostedService is to ensure all
-            // instrumentations, exporters, etc., are created and started.
-            Initialize(this.serviceProvider);
-        }
-        catch (Exception ex)
-        {
-            HostingExtensionsEventSource.Log.FailedOpenTelemetrySDK(ex);
-        }
+        // The sole purpose of this HostedService is to ensure all
+        // instrumentations, exporters, etc., are created and started.
+        Initialize(this.serviceProvider);
```

nit: add a comment saying intentionally not catching ex, to let it bubble up.